packages/app/src/index.d.ts

```diff
@@ -0,0 +1,550 @@
+/*
```

Comment:
if we are coding in typescript, there should be no commited .d.ts file anymore, should there? There should be types files in a dist/ dir, but those are derived and should not be committed to source, only rendered out during build for packaging + upload to npmjs on release I think?